### PR TITLE
VALIDATE #breadcat

### PR DESCRIPTION
**Pull request by Asteris92**
# Validate [bread](https://solscan.io/token/FG73ZjtfsfgzrTYKRbcsskXhVzWhUsPmBcV9MXH8VbYH)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/BreadCatCTO/status/1788573683809698271
- Coingecko/ CMC URL (If available): Coming soon!

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by Gizeh95 — COMMENTED** [review]
Good

**Review by Olicon0 — COMMENTED** [review]
Breadcat the best CTO project

**Review by Asteris92 — COMMENTED** [review]
Please check our metedate if they are correct.
[Metadata.txt](https://github.com/jup-ag/token-list/files/15264695/Metadata.txt)

**Review by Stusluv — COMMENTED** [review]
https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f6261666b726569626f72376863677a32357666707070616d646e6b3676796c34626a323773756965617477796465786834366a76743376787175342e697066732e6e667473746f726167652e6c696e6b

File: validated-tokens.csv [hunk 1]
```diff
@@ -1528,3 +1528,4 @@ cheepepe,CHEEPEPE,FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg,6,https://bafybei
 Nelson Moondela,$MOONDELA,BSXvX9dMvedcXUvwUQ5pzYJb9G8bBXKtddQns4oAxgEo,6,https://bafybeiebyum5ql7lhtgocj37hkesya762gt7vcno7shnykv2kvinbv6bu4.ipfs.cf-ipfs.com,true
 SHARE,SHARE,E6Eg7Esj5tfSwkbDGdrzhrotqptv7ghJNarLZ9rbHDSG,6,ipfs://bafkreihmnbzlnzod2x6abcoxyyqbm5mjccwihkvdjo2jwbxkztihpnhksa/,true
 Beluga Cat,BELUGA,E63CDwLy9Dwr3EptAzopV9RuWoQnn5ZVYEjLWnJX8dCw,9,https://bafybeid6xz4mx75na6wmjhn3axywvgs3d3gceie2evbg3koharbz4eptry.ipfs.nftstorage.link,true
+breadcat,bread,FG73ZjtfsfgzrTYKRbcsskXhVzWhUsPmBcV9MXH8VbYH,9,https://image-cdn.solana.fm/images/?imageUrl=https://bafkreibor7hcgz25vfpppamdnk6vyl4bj27suieatwydexh46jvt3vxqu4.ipfs.nftstorage.link,true
```
Comment (Asteris92) [inline validated-tokens.csv:1531]: 4a5cf57ecb99c26d621837f99da795bdc04e0695

**Review by Asteris92 — COMMENTED** [review]
4a5cf57ecb99c26d621837f99da795bdc04e0695

File: validated-tokens.csv (hunk 1 above)
Comment (Asteris92) [reply validated-tokens.csv:1531]: 4a5cf57ecb99c26d621837f99da795bdc04e0695

Comment (Asteris92) [inline validated-tokens.csv:1531]: 4a5cf57ecb99c26d621837f99da795bdc04e0695

**Comment by Jymfletzfur** [comment]
insufficient volume

**Comment by Asteris92** [comment]
Thank for your feed back, how much volume we need to be validated?

**Pull request closed without merging**